skimage/filters/tests/test_thresholding.py

```diff
@@ -321,8 +321,21 @@ def test_mean():
 
 
 def test_triangle_images():
-    assert(threshold_triangle(np.invert(data.text())) == 151)
-    assert(threshold_triangle(data.text()) == 104)
+    # Threshold uint8 image.
+    text = data.text()
+    assert(threshold_triangle(text) == 104)
+    int_bins = text.max() - text.min() + 1
+    # Threshold same image as float, setting nbins to match the integer case.
```

Comment:
Can do. Note that I'm taking the 'correct' thresholds for these sample images from the original test as ground truth, so a float test is really asking 'does it get the equivalent value as the uint test'?